build-and-test.ps1

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env pwsh
+[cmdletbinding()]
+param(
+    [ValidateSet('aspnet','wcf','samples')]
```

Comment:
For consistency, it would make sense to list "runtime" here but continue to default to "runtime" if it's not set.

Reply:
I debated a long time on this.  What do you think about renaming `manifest.json` to `manifest.runtime.json`?  The reason is there isn't a clear default anymore.

Reply:
I think it would help make things more clear and consistent.

Reply:
Good point.  Although now that I think about it, the "runtime" terminology isn't very fitting considering it also contains metadata for the SDK images.  What about "base"?

Reply:
My vote would be something more descriptive maybe - `manifest.runtime.sdk.json`  In this context `base` isn't meaningful to me.

Reply:
How about a very slight tweak: `manifest.runtime-sdk.json`

Reply:
Done